src/db/migrations/v2.2.0-update-opening-days-irregular-length.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.changeColumn("activities", "opening_days_irregular", {
+      type: Sequelize.STRING(1500)
+    });
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.changeColumn("activities", "opening_days_irregular", {
+      type: Sequelize.STRING
```

Comment:
are you sure this will reinstate default length? might it not just ignore since is string?

Reply:
Yes, I've verified against my local postgres database.